resources/src/main/res/layout/fragment_search.xml

```diff
@@ -10,7 +10,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginHorizontal="@dimen/default_margin"
-        android:layout_marginVertical="8dp"
+        android:layout_marginVertical="16dp"
```

Comment:
`@dimen/default_margin` is equal to `16dp`. Also now we are setting both horizontal and vertical margin to 16dp. So let's replace the specific properties and just use margin and set it to the dimen value.